modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java

```diff
@@ -103,9 +103,7 @@
   /** The engage base url property **/
   static final String ENGAGE_URL_PROPERTY = "org.opencastproject.engage.ui.url";
   /** The default path to the player **/
-  static final String DEFAULT_PLAYER_PATH = "/engage/ui/watch.html";
-  /** The configuration property value for the player location. */
-  static final String PLAYER_PROPERTY = "player";
+  static final String PLAYER_PATH = "/watch/";
 
```

Comment:
@lkiesow would you mind clarifying the comment to be something like "The default path to the engage-ui module's player REST endpoint", so future developer's know which module that path is linked to.